src/frontend/screens/Onboarding/Sync/index.tsx

```diff
@@ -30,7 +30,9 @@ const m = defineMessages({
   },
 });
 
-export const SyncOnboardingScreen: NavigationStackScreenComponent = ({}) => {
+export const SyncOnboardingScreen: NavigationStackScreenComponent<{
+  keepExistingObservations?: boolean;
```

Comment:
not used yet by the component but it'll eventually be needed to determine how the sync is conducted based on user preference